test/xpu_api/random/interface_tests/distributions_methods.pass.cpp

```diff
@@ -140,14 +140,14 @@ check_params(oneapi::dpl::extreme_value_distribution<T>& distr)
 }
 
 template <typename Distr>
-typename ::std::enable_if<::std::is_same<typename Distr::param_type,
-        ::std::pair<typename Distr::scalar_type, typename Distr::scalar_type>>::value, void>::type
+typename ::std::enable_if<::std::is_same<Distr, 

```

Comment:
While this is the test code I don't understand what is the purpose of those `enable_if`'s hear and on the line 184. Let's discuss